### DISCUSSION TITLE
how to enable rgw pod‘s cors，s3cmd mb s3://image  response with 405

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13555] [Q&A] [answered]
how to enable rgw pod‘s cors，s3cmd mb s3://image  response with 405

root@karmada-02-a:~# s3cmd mb  s3://images
ERROR: S3 error: 405 (MethodNotAllowed)

**Comment by zhucan** [comment]
what's pod's cors? cores?

**Reply by zhucan** [reply, marked as answer]
@13567436138 https://ceph-users.ceph.narkive.com/MY7lZUHe/using-radosgw-with-s3cmd-bucket-failure take a look at this doc, can help you?

**Reply by 13567436138** [reply]
solved my probleam
